src/main/java/com/manardenza/dao/HotelDaoImpl.java

```diff
@@ -18,6 +19,14 @@ public static HotelDaoImpl getInstance() {
         return instance;
     }
 
+    public List<Hotel> getHotelsByName(String hotelName) {
+        return null;
+    }
+
+    public List<Hotel> getHotelsByCity(String hotelCity) {
+        return new ArrayList<Hotel>();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). (sonar.java.source not set. Assuming 7 or greater.) [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS2293)